components/match_ticker/commons/match_ticker_display_components.lua

```diff
@@ -297,8 +298,14 @@ function Details:countdown(matchPageIcon)
 		:addClass('match-countdown')
 		:node(Countdown._create(countdownArgs))
 
-	if String.isNotEmpty(match.vod) then
-		countdownDisplay:node(VodLink.display{vod = match.vod})
+	if Logic.readBool(match.finished) then
+		local vods = Array.map(match.match2games, Operator.property('vod'))
+		if String.isNotEmpty(match.vod) then
+			table.insert(vods, match.vod)
```

Comment:
should the match vod maybe be the first one?

Reply:
Sure